CMakeLists.txt

```diff
@@ -529,7 +529,7 @@ configure_file(
 if(MRN_BUNDLED)
   set(MRN_DATA_DIR "${INSTALL_MYSQLSHAREDIR}/${PROJECT_NAME}")
 else()
-  set(MRN_DATA_DIR "share/${PROJECT_NAME}")
+  set(MRN_DATA_DIR "/usr/share/${PROJECT_NAME}")
```

Comment:
We should not hard-code the path.
This doesn't work with `-DCMAKE_INSTALL_PREFIX=/usr/local`.